src/lib/ruby-to-blocks-converter/gdx_for.js

```diff
@@ -20,6 +20,81 @@ const GdxForConverter = {
                     );
                 }
                 break;
+            case 'gdx_for_force':
+                if (args.length === 0){
+                    block = this._createBlock('gdxfor_getForce', 'value');
+                }
+                break;
+            case 'gdx_for_tilted?':
+                if (args.length === 1 && this._isString(args[0])) {
+                    block = this._createBlock('gdxfor_isTilted', 'value');
+                    this._addInput(
+                        block,
+                        'TILT',
+                        this._createFieldBlock('gdxfor_menu_tiltAnyOptions', 'tiltAnyOptions', args[0])
+                    );
+                }
+                break;
+            case 'gdx_for_tilt_angle':
+                if (args.length === 1 && this._isString(args[0])) {
+                    block = this._createBlock('gdxfor_getTilt', 'value');
+                    this._addInput(
+                        block,
+                        'TILT',
+                        this._createFieldBlock('gdxfor_menu_tiltOptions', 'tiltOptions', args[0])
+                    );
+                }
+                break;
+            case 'gdx_for_falling?':
+                if (args.length === 0) {
+                    block = this._createBlock('gdxfor_isFreeFalling', 'value');
+                }
+                break;
+            case 'gdx_for_spin_speed':
+                if (args.length === 1 && this._isString(args[0])){
+                    block = this._createBlock('gdxfor_getSpinSpeed', 'value');
+                    this._addInput(
+                        block,
+                        'DIRECTION',
+                        this._createFieldBlock('gdxfor_menu_axisOptions', 'axisOptions', args[0])
+                    );
+                }
+                break;
+            }
+        } else if ((this._isSelf(receiver) || receiver === Opal.nil) &&
+            name === 'when' &&
+            args.length === 2 && args[0].type === 'sym' &&
+            this._isString(args[1]) &&
```

Comment:
ここもなんだ。ためしにそのブロックのドロップダウンに変数のブロックを突っ込んでみてもらえるかな？
驚くことにドロップダウンにブロックを突っ込めるんだよね。

ということで、以下でお願いします。

```suggestion
            this._isStringOrBlock(args[1]) &&
```